### PR TITLE
Fixed rewards in YPP dashboard

### PR DESCRIPTION
**Pull request by attemka**

### CODE REVIEW

**Review by thesan — APPROVED** [review]

File: packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx [hunk 1]
```diff
@@ -98,20 +98,26 @@ export const YppDashboardMainTab: FC<YppDashboardMainTabProps> = ({ currentTier
...
-          const customMultiplier = reward.customMultiplier
+          const customMultiplier = reward.customMultiplier?.[currentTier]
           const rewardAmount = reward.joyAmount
             ? typeof reward.joyAmount === 'number'
               ? {
                   type: 'number' as const,
-                  amount: reward.joyAmount * (customMultiplier ? customMultiplier[currentTier] : multiplier),
+                  amount:
+                    reward.joyAmount === 0
+                      ? customMultiplier || multiplier
```
Comment (WRadoslaw) [inline packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx]: Why do we want to display the multiplier as an amount if there isn't any reward?

File: packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx [hunk 2]
```diff
@@ -98,20 +98,26 @@ export const YppDashboardMainTab: FC<YppDashboardMainTabProps> = ({ currentTier
...
               : { type: 'range' as const, min: reward.joyAmount.min, max: reward.joyAmount.max }
             : null
           const rewardAmountUsd = reward.usdAmount
             ? typeof reward.usdAmount === 'number'
               ? {
                   type: 'number' as const,
-                  amount: reward.usdAmount * (customMultiplier ? customMultiplier[currentTier] : multiplier),
+                  amount:
+                    reward.usdAmount === 0
+                      ? customMultiplier || 0
```
Comment (WRadoslaw) [inline packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx]: Same as above

File: packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx (hunk 1 above)
Comment (attemka) [reply packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx]: the thing is that this is used to display the rewards with the `baseAmount === 0`. The thing is that usually we'll do the `baseAmount * multiplier`, but in this case it'll result in all rewards will became 0. I don't want to create a separate var for that, but now I think that I can extend the type of baseReward to `number[]` and use the values without a multiplier if its that type. It seems that it'll be more obvious than to use the multiplier values.

Comment (attemka) [reply packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx]: @WRadoslaw please check the updated solution

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**